index.js

```diff
@@ -77,6 +77,17 @@ function getDefaultConfig(mainTemplate, templatesPath) {
         }
       };
 
+      // Add extra function for finding a security scheme name
+      ramlObj.securitySchemeName = function (name) {
```

Comment:
This sort of stuff should probably go into raml2obj, so that it will also work for raml2md and other future projects. It's also just nice to abstract this sort of stuff away from a client like raml2html. Do you think that makes sense?